PianoRoll.h

```diff
@@ -0,0 +1,516 @@
+/*
```

Comment:
This commit adds PianoRoll.h and PianoRoll.cpp to the top of the lmms src. Why?

Reply:
I cant understand what you mean? Are the files in different order?
I have only added code.

Reply:
This example is from your master branch. These two files are where they're supposed to be:
https://github.com/musikBear/lmms/blob/master/src/gui/editors/PianoRoll.cpp
https://github.com/musikBear/lmms/blob/master/include/PianoRoll.h

These are the duplicate ones that were added by you in 4492e2cb9932204c600333e0e219849f8cb735f3 :
https://github.com/musikBear/lmms/blob/master/PianoRoll.cpp
https://github.com/musikBear/lmms/blob/master/PianoRoll.h

This PR adds 5,734 lines of code. Does that sound like what you intended?

Reply:
> These are the duplicate ones that were added by you in 4492e2c :
https://github.com/musikBear/lmms/blob/master/PianoRoll.cpp
https://github.com/musikBear/lmms/blob/master/PianoRoll.h

oH! NOW i understand that weird thing!
I made a complete hickup, (spelled with an f) inside my *own* master-repo.
For some odd github reason i 'suddenly' had a whole new repo, instead of the commit i was trying to make. I had no idea that it could result in duplicate files, but yes i did see that weird message about 5734 lines of code.
I thought it was because i had used an external editor, and then deleted everything in the .h file, and *replaced* it with the content from the external editor, so i 'kind of' had changed everything. I asked on Discord what i should do. The reply was that 'it could happen'. 
I used the method 'Add-files', and they did not appear, but what i have done is to add the files to *root* of my master, instead of adding them inside the *sub-directories*!
The correct ones are :
* https://github.com/musikBear/lmms/blob/master/**src/gui/editors**/PianoRoll.cpp
* https://github.com/musikBear/lmms/blob/master/**include**/PianoRoll.h
The other are is garbage.
Github gives me some issues, because part of the ui is written in grey text, i then only see the black text, and then mistakes happens. 
Can you or can (should) I remove the two additional files?
-One question: What is 'Blob' ?

Reply:
As for what 'blob' is in this context. Dude, I don't know.
https://developer.github.com/v3/git/blobs/

>  Can you or can (should) I remove the two additional files?

Yes, those files must be removed. Since they were added in one distinct commit I would remove them by removing that commit. I would do this with the rebase command.
`git rebase -i HEAD~5` Will open the commits in your default text editor. Just delete the 
I would revert the commit with the rebase command. Just delete the line with commit 4492e2c and save/close the file.
Stage two is to `push -f`

Reply:
I will see if i can swing that. I am *not* strong in GIT-cmds, but as long as i am only messing with my own repo, i guess nothing serious bad can happen(?)

Reply:
The suggested action above removes a commit so it's destructive by design. Definitely seriously bad things can happen from this. Always make a backup of the branch you're working on if you're doing something you've never done before.

Reply:
Ok! Thanks for the heads-up!